scripts/entrypoint.sh

```diff
@@ -35,6 +35,15 @@ log_in_json()
     printf '{"level":"%s","ts":"%s","caller":"%s","msg":"%s"}\n' "$LOGTYPE" "$TIMESTAMP" "$FILENAME" "$MSG"
 }
 
+validate_env_var()
+{
+    log_in_json info "Validating env variables ..."
+    if [[ "$AWS_VPC_K8S_PLUGIN_LOG_FILE" == "stdout" ]]; then
```

Comment:
You may want to convert the input to lower to cover for upper/camel case or if there's a typo